codecov.yml

```diff
@@ -3,6 +3,9 @@
 coverage:
   # https://docs.codecov.com/docs/commit-status
   status:
+    patch:  
+      default:      
+        threshold: 2%
```

Comment:
Why this change?

Reply:
Chasing a 0.01% patch change seemed like a waste of time. I'm not sure what we want as a threshold, but essentially this was my idea.